src/stores/user-store.js

```diff
@@ -66,7 +85,17 @@ class UserStore extends EventEmitter {
         }
         else if (action.ACTION_TYPE === ACTION_TYPE.updateUser) {
             this._updateUser(action.user);
-            this.emit(EVENT_TYPE.change);
+            if (action.searchMode) {
```

Comment:
We can spit up logic with just UI `wantFind` setting and the mode of the whole application `searchMode`.
On the first click on the `Find` button we set `this._searchedUser` that switch on `searching mode` for our application. In this case we can get rid of the second arg in  the `updateUser` action.
On the second click (click on the `Stop searching` button) we should reset `this._searchedUser` in the store, it will switch off searching mode.
What do you think about it?